backend/src/replayUtils.ts

```diff
@@ -41,6 +43,7 @@ export class ReplayUtils {
       this.replayQueue = [];
       const answers: Environment.Adapter.Answers = this.replayStack.pop();
       ReplayUtils.setDefaults(questions, answers);
+      this.replayStack.splice(this.replayStack.length - this.numOfSteps +1);
```

Comment:
please explain why do we need + 1

Reply:
consider we are at step 3 // replayStack = ["Jan", "Feb", "March"];
When clicking 'Next' the replayStack is pushed by 1 obj. //  replayStack = ["Jan", "Feb", "March", "April"];
Then if clicking 2 steps back  // replayStack = ["Jan", "Feb"];
You are at step 2 now. and didn't click next yet. so replay stack should look // replayStack = ["Jan"]; = replayStack.splice(2 - 2 +1); = replayStack.splice(1);